core/templates/components/button-directives/social-buttons.component.html

```diff
@@ -26,6 +26,12 @@
         <span class="oppia-icon-accessibility-label">Twitter</span>
       </a>
     </div>
+    <div class="oppia-github-follow">
+      <a href="https://github.com/oppia/oppia" target="_blank" rel="noopener">
```

Comment:
I think we should use `https://github.com/oppia/` (better landing page) which contains readable information about the code and project. 

Also, the "Follow" button is on `https://github.com/oppia/` page!

Reply:
I have changed the github link to the main landing page: https://github.com/oppia/